app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java

```diff
@@ -213,6 +213,43 @@ public Course getCourse(int id) {
         return course;
     }
 
+    // Get courses by list of ids
+    public List<Course> getCourses(String[] ids) {
```

Comment:
i save it in sharedpreferences when the user adds a course. So we can fetch all the courses in the user's sharedpreferences at once